docs/resources/Message.md

```diff
@@ -345,6 +346,21 @@ There are multiple message types that have a `message_reference` object.
 - These messages have `message_id`, `channel_id`, and `guild_id`.
 - These messages will never have content, embeds, or attachments, mainly just the `message_reference` and `referenced_message` fields.
 
+###### Poll result messages
+
+- These are automatic messages sent after a poll has ended and the results have been finalized. (type 46)
+- These messages have `message_id` and `channel_id`, which point to the original poll message. The `channel_id` will be the same as that of the poll.
+- The author will be the same as the author of the poll and will be mentioned.
+- These messages contain a `poll_result` embed that can have the following fields:
+  - `poll_question_text` contains the question text from the original poll message.    
+  - `victor_answer_votes` contains the number of votes for the answer(s) with the most votes.
+  - `total_votes` contains the total number of votes in the poll.
+  - `victor_answer_id` contains the id for the winning answer, if applicable. 
+  - `victor_answer_text` contains the text for the winning answer, if applicable.
+  - `victor_answer_emoji_id` contains the id for the emoji associated with the winning answer, if applicable.
+  - `victor_answer_emoji_name` contains the name of the emoji associated with the winning answer, if applicable.
+  - `victor_answer_emoji_animated` specifies whether the emoji associated with the winning answer is animated, if applicable.         
```

Comment:
should we put this in a table so it's more readable?

Reply:
Would be cleaner. Yes please 🙏

Reply:
yeah this part about the embed fields should be in a new section in the embed object documentation. and we should add the automod embeds there later

Reply:
sorry, i've been on vacation but just made some updates based on this feedback. i do agree that the specific embed fields for each type warrants a new section. i didn't want to take on documenting fields for some of the other embed types here, but i can at least get it started and we can flesh this out in another PR.